docs/csharp/whats-new/breaking-changes.md

```diff
@@ -3,17 +3,18 @@ title: Breaking changes in the C# compiler
 description: Find any breaking changes in the C# compiler that you are using.
 ms.topic: troubleshooting
 ms.custom: updateeachrelease
-ms.date: 08/14/2020
+ms.date: 03/09/2021
 ---
 
 # Learn about any breaking changes in the C# compiler
 
 The [Roslyn](https://github.com/dotnet/roslyn) team maintains a list of breaking changes in the C# and Visual Basic compilers. You can find information on those changes at these links on their GitHub repository:
 
-- [Breaking changes in VS2019 version 16.8 which will be introduced for .NET 5.0 and C# 9.0](https://github.com/dotnet/roslyn/blob/master/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%205.md)
-- [Breaking changes in VS2019 Update 1 and beyond compared to VS2019](https://github.com/dotnet/roslyn/blob/master/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20post%20VS2019.md)
-- [Breaking changes since VS2017 (C# 7)](https://github.com/dotnet/roslyn/blob/master/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20post%20VS2017.md)
-- [Breaking changes in Roslyn 3.0 (VS2019) from Roslyn 2.* (VS2017)](https://github.com/dotnet/roslyn/blob/master/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20VS2019.md)
-- [Breaking changes in Roslyn 2.0 (VS2017) from Roslyn 1.* (VS2015) and native C# compiler (VS2013 and previous).](https://github.com/dotnet/roslyn/blob/master/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20VS2017.md)
-- [Breaking changes in Roslyn 1.0 (VS2015) from the native C# compiler (VS2013 and previous).](https://github.com/dotnet/roslyn/blob/master/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20VS2015.md)
-- [Unicode version change in C# 6](https://github.com/dotnet/roslyn/blob/master/docs/compilers/CSharp/Unicode%20Version.md)
+- [Breaking changes in Roslyn after .NET 5](https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20post%20DotNet%205.md)
+- [Breaking changes in VS2019 version 16.8 which will be introduced for .NET 5.0 and C# 9.0](https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%205.md)
```

Comment:
Let's change the tense here:

```suggestion
- [Breaking changes in VS2019 version 16.8 introduced with .NET 5.0 and C# 9.0](https://github.com/dotnet/roslyn/blob/main/docs/compilers/CSharp/Compiler%20Breaking%20Changes%20-%20DotNet%205.md)
```

Reply:
@BillWagner I had this the same as the title in Roslyn, but I agree past tense is more appropriate.